Kitos_PubSub.sln

```diff
@@ -3,29 +3,35 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.0.31903.59
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PubSub.Application", "PubSub.Application\PubSub.Application.csproj", "{80E096AD-6FFA-48F3-B3A5-4CF016C8FBF4}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PubSub.Application.Api", "PubSub.Application.Api\PubSub.Application.Api.csproj", "{80E096AD-6FFA-48F3-B3A5-4CF016C8FBF4}"
```

Comment:
There might be some leftover project references
![image](https://github.com/user-attachments/assets/50396d8d-85fe-488d-b640-1cdf504b31fe)